build.assets/go-modules/print-api-import-path/main.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2022 Gravitational, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Command print-api-import-path prints the import path that
+// should appear in Go import paths to stdout.
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/gravitational/teleport/build.assets/go-modules"
+	"github.com/gravitational/trace"
+)
+
+// prints the import path of the api module
+func main() {
+	modPath, err := modules.GetImportPath("./api")
```

Comment:
Could we ditch the hardcoded path and get it from os.Args[1] instead?

In this case the invocation becomes something like `go run ./build.assets/go-modules/print-api-import-path ./api`.